### PR TITLE
add metadataUrl

### PR DESCRIPTION
**Pull request by disassembler**

### CODE REVIEW

File: cardano-lib/default.nix [hunk 1]
```diff
@@ -62,6 +63,7 @@ let
       useByronWallet = true;
       relaysNew = "relays.staging.cardano.org";
       smashUrl = "https://smash.staging.cardano.org";
+      metadataUrl = "https://metadata.cardano-testnet.iohkdev.io";
```
Comment (jbgi) [inline cardano-lib/default.nix:66]:
```suggestion
      metadataUrl = "https://metadata.staging.cardano.org";
```

File: cardano-lib/default.nix [hunk 2]
```diff
@@ -135,6 +138,7 @@ let
       private = false;
       relaysNew = "relays-new.shelley-qa.dev.cardano.org";
       smashUrl = "https://smash.shelley-qa.dev.cardano.org";
+      metadataUrl = "https://metadata.cardano-testnet.iohkdev.io";
```
Comment (jbgi) [inline cardano-lib/default.nix:141]:
```suggestion
      metadataUrl = "https://metadata.shelley-qa.dev.cardano.org";
```

**Review by jbgi — CHANGES_REQUESTED** [review]

**Review by jbgi — APPROVED** [review]

**Review by nikolaglumac — APPROVED** [review]

**Comment by nikolaglumac** [comment]
@jbgi @disassembler can we please have this one merged? 🙏

**Comment by nikolaglumac** [comment]
We have already integrated this into Daedalus: https://github.com/input-output-hk/daedalus/pull/2292/commits/79517144f1d35e0c0f6f85d16866b1cdef7cd299

**Pull request merged**